chain-storage/benches/storage.rs

```diff
@@ -1,8 +1,7 @@
 use criterion::{criterion_group, criterion_main, BatchSize, Criterion};
 use rand_core::{OsRng, RngCore};
 
-use chain_core::property::Block as _;
-use chain_storage::{test_utils::Block, BlockStoreBuilder};
+use chain_storage::{test_utils::Block, BlockInfo, BlockStore};
```

Comment:
Is `test_utils` part of the production library, or should it be cfg-gated for tests?

Reply:
Not a part of production. It is enabled for `cfg(test)` and `with-bench` feature.

Reply:
Ah right, this is in benches.